src/core/ProgressBar/base.ts

```diff
@@ -0,0 +1,3 @@
+import { LinearProgressProps } from '@material-ui/core/LinearProgress';
+
+export type BaseProgressProps = Omit<LinearProgressProps, 'value' | 'color'>;
```

Comment:
This base file isn't required since value and color are being used in our component so no requirement of Omitting the same

Reply:
@S-ayanide THat means, I need to remove the base file?